ekwipunek.py

```diff
@@ -0,0 +1,823 @@
+    # a nie cokolwiek < - jest to Twój komentarz z review -> rozwiń proszę,
+    # ponieważ nie do końca rozumiem o co chodzi, jakie konkretne kwargsy w tym wypadku?
+    # myślę, że można to zawrzec również w efekt... Daj znać.
+    def __init__(self, nazwa: str, wartosc: int, efekt: dict[str, int] | None):
+        super().__init__(nazwa, wartosc, efekt)
+
+
+class Bronie(dict):
+    """
+    Kontener na przedmioty klasy Broń posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Pancerze(dict):
+    """
+    Kontener na przedmioty klasy Pancerz posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class PrzedmiotyMagiczne(dict):
+    """
+    Kontener na przedmioty klasy Magia posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Pisma(dict):
+    """
+    Kontener na przedmioty klasy Pisma posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Artefakty(dict):
+    """
+    Kontener na przedmioty klasy Artefakt posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Zywnosc(dict):
+    """
+    Kontener na przedmioty klasy Jedzenie posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Pozostalosci(dict):
+    """
+    Kontener na przedmioty klasy Pozostale posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+class Ekwipunek:
+    """
+    Klasa Ekwipunek -> To w niej znajdują się Przedmioty z Podłogi dodane za pomocą
+    Bohatera. Tutaj również znajduje się kontener w uzyciu (nie wiem, czy słusznie).
+    :param brone: kontener na przedmioty klasy Broń
+    :param pancerze: kontener na przedmioty klasy Pancerz
+    :param przedmioty magiczne: kontener na przedmioty klasy Magia
+    :param pisma: kontener na przedmioty klasy Pismo
+    :param artefakty: kontener na przedmioty klasy Artefakt
+    :param jedzenie: kontener na przedmioty klasy Jedzenie
+    :param pozostale: kontener na przedmioty klasy Pozostale
+    :param magazyn: słownik zawierający wszystkie kontenery przypisane
+        do odpowiedniego klucza
+    :param w_uzyciu: kontener zawierający przedmioty znajdujące się w danej
+        chwili w użyciu
+    """
+    def __init__(self):
+        self._bronie = Bronie()
+        self._pancerze = Pancerze()
+        self._przedmioty_magiczne = PrzedmiotyMagiczne()
+        self._pisma = Pisma()
+        self._artefakty = Artefakty()
+        self._jedzenie = Zywnosc()
+        self._pozostale = Pozostalosci()
+        self._magazyn = {
+            "Bronie": self._bronie,
+            "Pancerze": self._pancerze,
+            "Przemioty Magiczne": self._przedmioty_magiczne,
+            "Pisma": self._pisma,
+            "Artefakty": self._artefakty,
+            "Jedzenie": self._jedzenie,
+            "Pozostałe": self._pozostale,
+        }
+        self._w_uzyciu = {}
```

Comment:
To jest okej, bo jakoś trzeba oznaczyć przedmiot. Dodaj dodatkowy/e atrybuty które określają maksymalną ilość założonych przedmiotów w zależności od klasy

Reply:
To ma być atrybut przedmiotu (np jedna broń) czy atrybuty ekwipunku?

Reply:
### MAJA TO BYC STALE ZHARDKODOWANE W KLASIE EKWIPUNKU